tests/virt_autotest/virt_autotest_base.pm

```diff
@@ -186,10 +201,16 @@ sub upload_virt_logs {
     my ($log_dir, $compressed_log_name) = @_;
 
     my $full_compressed_log_name = "/tmp/$compressed_log_name.tar";
+    if (check_var('ARCH', 's390x')) {
+        type_string "tar cvf $full_compressed_log_name $log_dir; gzip -f $full_compressed_log_name; rm $log_dir -r\n";
```

Comment:
It would be better to use "bash_cmd1 && bash_cmd2 && bash_cmd3" in type_string if bash_cmd2 depends on successful return of bash_cmd1 and so on.